Remove CodeScene Dockerfile

knorrest opened the pull request.

Commit:
Remove CodeScene Dockerfile

jumarko commented:
Is there a new place where people can find a complete example of how to build a CodeScene docker image from scratch?
Is all relevant info documented elsewhere?
Did you update the on-prem docs?

knorrest commented:
We provide a docker image and information about how to use it. I see no reason why we should provide a complete example of how to build a custom docker image. Anyone wanting to do that must know docker and should be able to do it with the information on how to run the jar in that is in the docs.
I added a docker-README to the onprem repo and added it to the image on docker hub (we should eventually update it from builds when pushing the image)
Good point about the on-prem docs. I find only one place that mention the dockerfile, will update it to say docker image and link to docker hub.

jumarko commented on docker-codescene/start-codescene.sh:
```diff
@@ -1,7 +0,0 @@
-mkdir -p $CODESCENE_ANALYSIS_RESULTS_ROOT
-mkdir -p $CODESCENE_CLONED_REPOSITORIES_ROOT
-# don't quote "$JAVA_OPTIONS" because you wouldn't be able to use it for multiple properties/settings in docker-compose.yml
-java -XshowSettings:vm $JAVA_OPTIONS -XX:MaxRAMPercentage=75 -Duser.timezone="$CODESCENE_TIMEZONE" -jar "/opt/codescene/codescene-enterprise-edition.standalone.jar" | tee -a ${CODESCENE_DIR}/codescene.log
```
This wasn't reflected in the README. E.g. `-XX:MaxRAMPercentage` is missing (there are only [outdated instructions for old version of JDK 8](https://github.com/empear-analytics/docker-codescene-nginx-self-signed-ssl#memory-settings))

knorrest replied:
Not sure what you mean - should I delete the outdated instructions?
The MaxRAMPercentage setting is included in our docker image. I we should write anything about that setting we should do it in the docker hub readme.

jumarko commented on README.md:
```diff
@@ -31,15 +31,14 @@ The reverse proxy using Nginx is built like this:
 
-    docker build  --build-arg CODESCENE_VERSION=3.X.Y -t empear/ubuntu-onprem docker-codescene/
-	
 ## Run
```
There's a link in the README to https://github.com/empear-analytics/docker-codescene-nginx-self-signed-ssl/blob/master/docker-codescene/codescene which no longer exists. 
There are likely a few other broken links and source code listings.

knorrest replied:
I tried to update the README, but apparently missed this reference. Thx
